build.gradle

```diff
@@ -24,7 +24,7 @@ apply plugin: 'com.github.sherter.google-java-format'
 
 group = "com.google.api"
 archivesBaseName = "api-common"
-version = "0.0.3-SNAPSHOT"
```

Comment:
Maybe do 0.1.0 now - this repo basically has code moved from other places that is relatively stable.